Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionCiscoMeraki.yaml

```diff
@@ -143,6 +143,8 @@ ParserQuery: |
     let DvcActionLookup = datatable(pattern: string, DvcAction: string, EventResult: string)[
         "allow", "Allow", "Success",
         "deny", "Deny", "Failure",
+        "0", "Allow", "Success",
+        "1", "Deny", "Failure",
```

Comment:
@tduarte14 can you please attach sample log where these 'pattern' values are coming as 0 and 1.

Reply:
I found this from the documentation you shared.
948077334.886213117 MX60 flows src=39.41.X.X dst=114.18.X.X protocol=udp sport=13943 dport=16329 pattern: 1 all

Reply:
yes that's the one that shows it. My sample is similar to that one